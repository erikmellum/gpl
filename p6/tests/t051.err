Semantic error on line 1: Incorrect type for parameter 'x' of object r.
