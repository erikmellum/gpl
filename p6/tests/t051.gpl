rectangle r(x = 3.145);
