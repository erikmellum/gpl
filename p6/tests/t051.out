gpl.cpp::main()
  input file(tests/t051.gpl)
  random seed(42)
  read_keypresses_from_standard_input(false)
  dump_pixels(false)
  symbol_table(true)
  print_symbol_table(true)
  graphics(true)

gpl.cpp::main() Calling yyparse()


gpl.cpp::main() after call to yyparse().

1 error found.
gpl giving up.
